api/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
     groovy
 }
 
-val graphqlJavaVersion = "0.0.0-2022-01-11T23-37-10-2d522ede"
+val graphqlJavaVersion = "0.0.0-2022-02-07T22-39-39-39451d5e"
```

Comment:
This brings in a validation performance fix that added a nice chunk of performance improvements